### ISSUE TITLE
Segmentation of somas from oblongated cells

### ISSUE THREAD

**Issue by Antorithms** [MouseLand/cellpose#394]
Hi, 
I have trying out cellpose to segment somas from neurons labelled with completely cell filling RFP or GFP. From a few tests it looks like (i think because of the absence of a non-labelled nucleus) the cyto model performs very poorly. On the other hand the nuclei one does amazingly well when presented with small ROIs within a section but then fails strikingly when it is instead presented with the entire section, where many areas actually have no cells. I was willing to provide some training data to segment these images and wanted your advice on - 1 how to do that and - 2 if I should do it on the nuclei model. Maybe given the very specific and rather non-general use of the nuclei model maybe I could add some training data to it to generate something like a new model for my own use or for everyone to segment the somas of completely filled oblongated cells.

Thanks a lot in advance for your help!!

**Comment by kevinjohncutler** [comment]
@Antorithms could you share some example images? That is interesting that the behavior is so different depending on the FoV, but it might make sense depending on the image and cell sizes in pixels.

**Comment by Antorithms** [comment]
Hi Kevin, 
Here you can see two examples which are overall representative of the data we have. We are working around an experimental solution to simplify counting but using cellpose would be majorly useful for our purpose!
Thanks a lot and sorry for the delay in replying!

<img width="321" alt="image" src="https://user-images.githubusercontent.com/87311272/148528435-9f179162-8fbb-4bd9-a594-d728ab2e433d.png">

<img width="307" alt="image" src="https://user-images.githubusercontent.com/87311272/148528682-499929d0-6b63-4837-8100-21642e041789.png">

**Comment by kevinjohncutler** [comment]
@Antorithms sorry your issue got buried, I wasn't notified of your reply! These really long and thin cells are exactly what Omnipose was designed to address, and we were actually looking for a good ground-truth dataset of neurons but couldn't find one. I'd be really interested to see what your expected segmentation should look like for this data. The cell overlaps would be a big problem, but if all you care about is cell counting, you can exclude the long protrusions of the cell and just label the main body. Then Cellpose/Omnipose would just be a glorified 'blob detector', but it would get the job done!
